flutter_local_notifications/example/lib/windows.dart

```diff
@@ -0,0 +1,391 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
+
+import 'padded_button.dart';
+import 'plugin.dart';
+
+const WindowsInitializationSettings initSettings = WindowsInitializationSettings(
+  appName: 'Flutter Local Notifications Example',
+  appUserModelId: 'Com.Dexterous.FlutterLocalNotificationsExample',
+  guid: 'd49b0314-ee7a-4626-bf79-97cdb8a991bb',
+);
+
+List<Widget> examples({
+  required TextEditingController xmlController,
+  required VoidCallback showXmlNotification,
+}) => <Widget>[
+  const Text('Windows-specific examples',
+    style: TextStyle(fontWeight: FontWeight.bold),
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show short and long notifications notification',
+    onPressed: () async {
+      await _showWindowsNotificationWithDuration();
+    },
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show different scenarios',
+    onPressed: () async {
+      await _showWindowsNotificationWithScenarios();
+    },
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show notifications with some detail',
+    onPressed: () async {
+      await _showWindowsNotificationWithDetails();
+    },
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show notifications with image',
+    onPressed: () async {
+      await _showWindowsNotificationWithImages();
+    },
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show notifications with columns',
+    onPressed: () async {
+      await _showWindowsNotificationWithGroups();
+    },
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show notifications with progress bar',
+    onPressed: () async {
+      await _showWindowsNotificationWithProgress();
+    },
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show notifications with dynamic content',
+    onPressed: () async {
+      await _showWindowsNotificationWithDynamic();
+    },
+  ),
+  PaddedElevatedButton(
+    buttonText: 'Show notitification with activation',
```

Comment:
notification has been spelt incorrectly here and in a few other places

Reply:
Whoops, fixed. FYI, apparently, my autocomplete suggested this because it found this spelling used in `ios/Classes/FlutterLocalNotifications.m`. I didn't change it in my commit, just thought you should know

Reply:
Thanks will take a look and address those outside of this PR :)